definitions/tools/vcf_sanitize.cwl

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Sanitize a VCF"
```

Comment:
Can we make the filename/label more specific? I'll link to the immuno PR as soon as I make it, but after our conversation a few days ago I also made a vcf sanitizing tool that fixes the illegal format tags (containing underscores) from mutect? Or maybe the two tools could be merged, since both handle issues that conflict with the VCF specification

Reply:
I agree with being more specific about what's going on here (I noted it below, but up top would be good too).  My vote is to merge the two scripts. Do you already have a PR for yours?

Reply:
I'm working on the summary now, it'll be ready soon

Reply:
See #751